site/en/blog/extension-manifest-converter/index.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Extension Manifest Converter
```

Comment:
This sounds like it only converts the manifest. Would "Manifest V3 extension converter" make more sense?

Also we use sentence case for titles, e.g. "Extension manifest converter" if we were keeping the title unchanged.

Reply:
Updated the title to use sentence case.